client/app/components/sections/SearchPage/SearchPage.js

```diff
@@ -0,0 +1,41 @@
+import { Component, PropTypes } from 'react';
+import { div, img, p } from 'r-dom';
+
+import css from './SearchPage.css';
+
+class SearchPage extends Component {
+
+  constructor(props, context) {
+    super(props, context);
+    this.listings = this.props.searchPage.get('listings') || [];
```

Comment:
The `this.listings` property is only used in the render. I don't think that's a reason to make this a class instead of a clean stateless functional component that should be the default choise.